### ISSUE TITLE
Linux Mint build errors from qtofflinegl.cpp

### ISSUE THREAD

**Issue by SumGaea** [opentoonz/opentoonz#913]
Did `$ LANG=C make VERBOSE=1`
Got to 4% built then failed with the following errors.
```
/usr/bin/cmake -E cmake_progress_report /home/user/opentoonz/toonz/build/CMakeFiles 
[  4%] Building CXX object tnzcore/CMakeFiles/tnzcore.dir/__/common/tvrender/qtofflinegl.cpp.o
cd /home/user/opentoonz/toonz/build/tnzcore && /usr/bin/c++   -DLINUX -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_OPENGL_LIB -DQT_SHARED -DQT_WIDGETS_LIB -DTCOLOR_EXPORTS -DTFLASH_EXPORTS -DTGEOMETRY_EXPORTS -DTGL_EXPORTS -DTIMAGE_EXPORTS -DTIMAGE_IO_EXPORTS -DTNZCORE_EXPORTS -DTNZ_LITTLE_ENDIAN=1 -DTRASTERIMAGE_EXPORTS -DTRASTER_EXPORTS -DTROP_EXPORTS -DTSOUND_EXPORTS -DTSTREAM_EXPORTS -DTSYSTEM_EXPORTS -DTVECTORIMAGE_EXPORTS -DTVRENDER_EXPORTS -Dtnzcore_EXPORTS -Wwrite-strings -Wundef -Wmissing-declarations  -fpermissive -lstdc++ -std=c++11 -O3 -DNDEBUG -fPIC -I/home/user/opentoonz/toonz/build/tnzcore -I/home/user/opentoonz/toonz/sources/tnzcore -I/home/user/opentoonz/toonz/sources/include -isystem /home/user/opentoonz/toonz/sources/BEFORE -isystem /home/user/opentoonz/thirdparty/tiff-4.0.3/libtiff -isystem /usr/include/libusb-1.0 -isystem /home/user/opentoonz/thirdparty/superlu/SuperLU_4.1/include/superlu -isystem /home/user/opentoonz/toonz/sources/tnzcore/../common/flash -isystem /home/user/opentoonz/thirdparty/Lz4/Lz4_131/lib -isystem /usr/include/freetype2 -I/usr/include/qt5 -I/usr/include/qt5/QtOpenGL -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -I/usr/include/qt5/QtNetwork    -o CMakeFiles/tnzcore.dir/__/common/tvrender/qtofflinegl.cpp.o -c /home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp
/home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp: In member function 'virtual void QtOfflineGL::createContext(TDimension, std::shared_ptr<TOfflineGL::Imp>)':
/home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp:180:36: warning: format '%x' expects argument of type 'unsigned int', but argument 3 has type 'Qt::HANDLE {aka void*}' [-Wformat=]
          QThread::currentThreadId());
                                    ^
/home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp: In member function 'virtual void QtOfflineGL::getRaster(TRaster32P)':
/home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp:228:63: error: no matching function for call to 'QOpenGLFramebufferObject::toImage(bool)'
                           (TPixelRGBM32 *)m_fbo->toImage(false).bits(), false));
                                                               ^
/home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp:228:63: note: candidate is:
In file included from /usr/include/qt5/QtGui/QtGui:51:0,
                 from /usr/include/qt5/QtOpenGL/QtOpenGLDepends:3,
                 from /usr/include/qt5/QtOpenGL/QtOpenGL:3,
                 from /home/user/opentoonz/toonz/sources/include/qtofflinegl.h:8,
                 from /home/user/opentoonz/toonz/sources/common/tvrender/qtofflinegl.cpp:3:
/usr/include/qt5/QtGui/qopenglframebufferobject.h:101:12: note: QImage QOpenGLFramebufferObject::toImage() const
     QImage toImage() const;
            ^
/usr/include/qt5/QtGui/qopenglframebufferobject.h:101:12: note:   candidate expects 0 arguments, 1 provided
make[2]: *** [tnzcore/CMakeFiles/tnzcore.dir/__/common/tvrender/qtofflinegl.cpp.o] Error 1
make[2]: Leaving directory `/home/user/opentoonz/toonz/build'
make[1]: *** [tnzcore/CMakeFiles/tnzcore.dir/all] Error 2
make[1]: Leaving directory `/home/user/opentoonz/toonz/build'
make: *** [all] Error 2
```
Ask away if more info needed.
Thanks.

**Comment by ideasman42** [comment]
Please post information on your system (distro version, Qt version... any other detail).
Just built OpenToonz on ArchLinux,  with QT-5.7, GCC-6.2.1.
it worked without problems

**Comment by SumGaea** [comment]
Distro: Mint 17.3 XFCE Rosa 64-bit
Kernel: 3.19.0-32

libsuperlu3-dev 3.0+20070106-3 (mentioning because libsuperlu-dev was specified in dependencies)

```
$ gcc --version
gcc (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4

$ qmake --version
QMake version 3.0
Using Qt version 5.2.1 in /usr/lib/x86_64-linux-gnu 

```

Apologies. Newest available package is older than 5.5
I'll have to figure out compilation then specifying it for the opentoonz make.

That or a newer distro.

**Comment by ideasman42** [comment]
Committed CMake check, to disallow building with unsupported Qt versions.
